composer.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "discoverygarden/islandora_collection_search",
+  "description": "Islandora Collection Search",
+  "type": "drupal-module",
+  "version": "7.x-dev"
```

Comment:
Not sure this is required given that I believe the version can be inferred via https://getcomposer.org/doc/04-schema.md#version?

Reply:
@jordandukart I've just checked it and you are totally right, it is not needed at all. I've just removed it and updated the PR. Thanks for your review!